engine/schema/src/main/java/com/cloud/dc/DataCenterVO.java

```diff
@@ -134,6 +135,9 @@
     @Column(name = "is_local_storage_enabled")
     boolean localStorageEnabled;
 
+    @Column(name = "sort_key")
```

Comment:
@rhtyd should we store such a enduser specific configuration in the business object for datacentre? It seems ouyt of place here. What's your consideration?

Reply:
Agree they can be details, however, this replicates the design/implementation for the template sorting feature; the sort_key is also used in template tables.